src/rnr_score_hv/innov_netcdf.py

```diff
@@ -0,0 +1,329 @@
+"""
+Copyright 2022 NOAA
+All rights reserved.
+
+Collection of methods to facilitate file/object retrieval
+
+"""
+from collections import namedtuple
+from dataclasses import dataclass, field
+from datetime import datetime
+import netCDF4
+
+from rnr_score_hv.config_base import ConfigInterface
+from rnr_score_hv import file_utils
+
+valid_metrics = [
+    'temperature',
+    'spechumid',
+    'uvwind'
+]
+
+VALID_STATS = [
+    'bias',
+    'count',
+    'rmsd'
+]
+
+
+@dataclass
+class Region:
+    ''' region object storing region name and min/max latitude bounds '''
+    name: str
+    min_lat: float
+    max_lat: float
+
+    def __post_init__(self):
+        if self.min_lat > self.max_lat:
+            msg = f'min_lat must be less than max_lat - min_lat: {self.min_lat}, '\
+                f'max_lat: {self.max_lat}'
+            raise ValueError(msg)
+        if self.max_lat < self.max_lat:
```

Comment:
i dont follow this "if" statement. isn't the left and right side of "<" is the same?

Reply:
Yeah, neither do I.  Not sure how that got in there.  I'll fix it.  Good catch!